orchestration/_tests/test_globus_flow.py

```diff
@@ -51,6 +51,123 @@ def __init__(self, root_path, uuid_value=None):
         self.uri = f"mock_endpoint_uri_{self.uuid}"
 
 
+# Mock the Client class to avoid real network calls
+class MockGlobusComputeClient:
+    def __init__(self, *args, **kwargs):
+        # No real initialization, as this is a mock
+        pass
+
+    def version_check(self):
+        # Mock version check to do nothing
+        pass
+
+    def run(self, *args, **kwargs):
+        # Return a mock task ID
+        return "mock_task_id"
+
+    def get_task(self, task_id):
+        # Return a mock task response
+        return {
+            "pending": False,
+            "status": "success",
+            "result": "mock_result"
+        }
+
+    def get_result(self, task_id):
+        # Return a mock result
+        return "mock_result"
+
+
+class MockSecret:
+    value = str(uuid4())
```

Comment:
I think it's best practice to have deterministic behavior for testing, but maybe there's a reason we're using a different one here each time. I don't think you can set the seed for uuid4 so maybe just pick a value and set it here?

if it needs to be real uuid4:
`value = "550e8400-e29b-41d4-a716-446655440000"`

if you need a string
`value = "my-mock-secret-acbd-1234"`